pegasus/sites.v3/code.org/public/educate/student-council.md.erb

```diff
@@ -1,60 +0,0 @@
-
----
-title: 2021-22 Code.org CSA Student Advisory Council
-video_player: true
-theme: responsive
-style_min: true
----
-
-<%=view :solid_block_header, :title=>"2021-22 Code.org CSA Student Advisory Council" %>
-
-<link href="/shared/css/course-blocks.css", type="text/css", rel="stylesheet"></link>
-
-<link href="/css/educate.css" rel="stylesheet">
-
-<style>
-  details summary {
-    cursor: pointer;
-  }
-</style>
-
-<br>
-
-<center><img src="/images/csa/student-council-2.jpg", width=100%></center>

```

Comment:
looks like you could also consider removing pegasus/sites.v3/code.org/public/images/csa/student-council-2.jpg

Reply:
Done! (might try to tackle the images folder next summer 😅)